Dockerfile-alpine.template

```diff
@@ -10,11 +10,13 @@ COPY requirements.txt /tmp/
 RUN set -ex \
     && apk add --no-cache --virtual .build-deps \
        gcc \
-       postgresql-dev=${PG_VERSION} \
+       postgresql-dev=$PG_VERSION \
        musl-dev \
     && apk add --no-cache \
        gettext \
-       postgresql-client=${PG_VERSION} \
+       postgresql-client=$PG_VERSION \
+    && apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
+        libressl2.7-libcrypto \    
```

Comment:
This is confusing, but appears to be a required upgrade from 2.6 given that `gdal` is coming from `edge`. I wasn't able to make it work with another combination. 😞

Reply:
Yeah, this was my experience as well. 

Want to resurface these:

 - appropriate/docker-postgis#96
 - appropriate/docker-postgis#97